app/src/main/res/values/dimens.xml

```diff
@@ -44,6 +44,13 @@
     <dimen name="font_large_extra">20sp</dimen>
     <dimen name="font_big">22sp</dimen>
     <dimen name="font_huge">24sp</dimen>
+    <dimen name="image_view_height">180dp</dimen>
+    <dimen name="view_height">60dp</dimen>
+    <dimen name="margin">16dp</dimen>
+    <dimen name="button_height">40dp</dimen>
+    <dimen name="icon_size">40dp</dimen>
+    <dimen name="zero">0dp</dimen>
```

Comment:
Zero dimen is not really necesary, we made an exception for this case in layouts. Every custom dimen, please move it below "Add your custom dimens here"